vendor/bat-native-ledger/src/bat/ledger/internal/bitflyer/bitflyer_authorization.cc

```diff
@@ -170,19 +171,17 @@ void BitflyerAuthorization::OnAuthorize(
                                    base::Unretained(this), std::move(callback),
                                    std::move(token), std::move(address));
 
-  if (RequestFor<PostConnectBitflyer> request{ledger_,
-                                              std::move(linking_info)}) {
-    std::move(request).Send(std::move(on_connect));
-  } else {
-    std::move(on_connect).Run(mojom::Result::LEDGER_ERROR);
-  }
+  RequestFor<PostConnectBitflyer>{ledger_, std::move(linking_info)}.Send(
```

Comment:
Nit: I think this [style rule](https://chromium.googlesource.com/chromium/src/+/master/styleguide/c++/c++-dos-and-donts.md#variable-initialization) suggests using parens for constructors.

Reply:
Dang, Chromium seems to be a bit overconcerned about variable initialization...
In contrast, [here](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax)'s what the C++ Core Guidelines has to say about it — amending regardless.